src/System.Diagnostics.TraceSource/tests/TraceEventCacheClassTests.cs

```diff
@@ -53,7 +53,6 @@ public void CallstackTest()
         {
             var cache = new TraceEventCache();
             Assert.Contains("at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)", cache.Callstack);
-            Assert.Contains("at System.Environment.get_StackTrace()", cache.Callstack);
```

Comment:
It should be really other way around: `Environment.GetStackTrace` should not be there because of it is internal method. On the other hand, it would be reasonable to make sure that `System.Environment.get_StackTrace` is included in the string because of it was always like that.

Related to https://github.com/dotnet/coreclr/issues/6209 .